### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by gsrajpal**
Error string bug fix

### CODE REVIEW

**Review by elmiko — APPROVED** [review]
/approve
/lgtm

**Comment by elmiko** [comment]
@JoelSpeed ptal

File: pkg/actuators/client/client.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ import (
 	ibmcloudproviderv1 "github.com/openshift/cluster-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1beta1"
 )
 
+// constants declarations
+const (
+	instanceNotFoundError string = "instance not found"
+)
```
Comment (JoelSpeed) [inline pkg/actuators/client/client.go:31]: Any reason not to make this an error type, eg 

```
instanceNotFoundError = errors.New("instance not found")

...

if errors.Is(err, instanceNotFoundError) {
```

**Review by elmiko — APPROVED** [review]
/lgtm

**Pull request merged**